test_project/sample/models.py

```diff
@@ -116,3 +116,9 @@ class RelatedToItemViaPrimaryKey(models.Model):
 
 class RelatedToItemViaUniqueField(models.Model):
     item = models.ForeignKey(Item, to_field="legacy_id", on_delete=models.CASCADE)
+
+
+class RelatedToItemAssignDb_column(models.Model):
+    item = models.ForeignKey(
+        Item, db_column="item", to_field="legacy_id", on_delete=models.CASCADE
```

Comment:
Why do you specify `db_column` value if you intent to use the same name as what django would generate per default ?

This fixes it.
```suggestion
        Item, db_column="item_", to_field="legacy_id", on_delete=models.CASCADE
```
This. fixes it also, and the db_column is still "item"
```suggestion
        Item, to_field="legacy_id", on_delete=models.CASCADE
```

I'm trying to understand the motivation of specifying the foreign key like you did.

Reply:
> Why do you specify `db_column` value if you intent to use the same name as what django would generate per default ?
> 
> This fixes it.
> 
> This. fixes it also, and the db_column is still "item"
> 
> I'm trying to understand the motivation of specifying the foreign key like you did.

For the foreign key, the database column name generated by Django by default is different from the attribute name of the model. The database column name generated by Django will be added _id

Reply:
> Why do you specify `db_column` value if you intent to use the same name as what django would generate per default ?

For the foreign key, the database column name generated by Django by default is different from the attribute name of the model. The database column name generated by Django will be added _id


> 
> This fixes it.
> 
> This. fixes it also, and the db_column is still "item"
> 
> I'm trying to understand the motivation of specifying the foreign key like you did.

I want to change the field types to foreign key without changing the corresponding database column name

Reply:
aldjemy knows that `item` will become `item_id` on the db, because it uses `field.column` as source of truth. which will be `'item_id'` if you don't specify `column`

Reply:
> aldjemy knows that `item` will become `item_id` on the db, because it uses `field.column` as source of truth. which will be `'item_id'` if you don't specify `column`

Add a rule: when the attribute name of the model is the same as db_column, the relationship generated by aldjemy is suffixed with _rs,This fixes it.